stumpy/mstumped.py

```diff
@@ -58,63 +58,55 @@ def mstumped(dask_client, T, m):
     See mSTAMP Algorithm
     """
 
-    hosts = list(dask_client.ncores().keys())
-    nworkers = len(hosts)
+    T_A = np.asarray(core.transpose_dataframe(T)).copy()
+    T_B = T_A.copy()
```

Comment:
I am a bit confused as to why we need T_B? It doesn't look like its being used for anything

Reply:
I wanted to implement nan/inf handling the same way as we did in `stump`. There you first calculate the `mean/stddev` and for each subsequence to ignore you set the mean to `inf`. Then you set `T_A=0` everywhere, where it was `nan`. Then you calculate the first matrix profile value, and here `T_B` can still be `nan`, because this information has to be passed to `core.mass`. Afterwards you set `T_B` to zero everywhere where it was `nan` and pass those two altered timeserieses to `_stump`, where the decision if one subsequence should be ignored is made only by looking at the mean and checking if it is `inf`.

So actually to support AB-joins we only need to pass `T_B` to `_mstump` and handle it from there, but I skipped this as this is a completely different issue.

Basically, at the moment `T_B` is only used to correctly calculate the first matrix profile value.